tests/shared/src/test/scala/cats/tests/EitherSuite.scala

```diff
@@ -369,6 +373,22 @@ class EitherSuite extends CatsSuite {
     assert(either.toEitherNel === (Either.right[NonEmptyList[String], Int](42)))
   }
 
+  test("liftLeftTo Left") {
```

Comment:
`leftLiftTo` or `liftLeftTo` – that's a tricky question :)

Reply:
Yeeeah:) I kept automatically writing `liftLeftTo`, but `leftLiftTo` is more consistent with other methods e.g. `leftWiden`. Fixed:)